aopy/analysis.py

```diff
@@ -74,19 +74,21 @@ def factor_analysis_dimensionality_score(data_in, dimensions, nfold, maxiter=100
 
 
 # These functions are for curve fitting and getting modulation depth and preferred direction from firing rates
-def func(target, b1, b2, b3):
+def tuning_fit_func(target, b1, b2, b3):
```

Comment:
you changed the name of func to tuning_fit_func, but you forgot to change it in run_curvefitting. a good example of why unit tests are important!